### PR TITLE
[Automated] Apply Daily Pull Request automatic update

### PR DESCRIPTION
**Pull request by samsonasik**
per https://github.com/rectorphp/rector-src/pull/226#issuecomment-862674086

### CODE REVIEW

File: .github/workflows/daily_pull_request_automatic_update.yaml [hunk 1]
```diff
@@ -50,3 +49,11 @@ jobs:
                     branch: ${{ matrix.actions.branch }}
                     title: '[automated] ${{ matrix.actions.name }}'
                     delete-branch: true
+
+            - name: Enable Pull Request Automerge
+                if: steps.cpr.outputs.pull-request-operation == 'created'
+                uses: peter-evans/enable-pull-request-automerge@v1
+                with:
+                token: ${{ secrets.ACCESS_TOKEN }}
```
Comment (samsonasik) [inline .github/workflows/daily_pull_request_automatic_update.yaml:57]: it needs personal access token as documented at https://github.com/peter-evans/enable-pull-request-automerge#action-inputs

**Comment by samsonasik** [comment]
@TomasVotruba let's try this.

**Comment by samsonasik** [comment]
It seems the GHA "ECS Expected — Waiting for status to be reported" keep loading, @TomasVotruba could you merge it ? Thank you.

**Comment by TomasVotruba** [comment]
> It seems the GHA "ECS Expected — Waiting for status to be reported" keep loading, @TomasVotruba could you merge it ? Thank you.

I had to rename action in required branches. All good now :+1:

**Comment by TomasVotruba** [comment]
Thank you :+1:

**Pull request merged**